lib/models/application.ts

```diff
@@ -206,33 +207,62 @@ const getApplicationModel = function (
 		 * 	console.log(applications);
 		 * });
 		 */
-		async getAll(options?: PineOptions<Application>): Promise<Application[]> {
-			if (options == null) {
-				options = {};
-			}
-
+		async getAll(
+			options?: PineOptions<Application>,
+			context?: 'directly_accessible',
+		): Promise<Application[]> {
 			const apps = await pine.get({
 				resource: 'application',
 				options: mergePineOptions(
 					{
-						$filter: {
-							is_directly_accessible_by__user: {
-								$any: {
-									$alias: 'dau',
-									$expr: {
-										1: 1,
+						...(context === 'directly_accessible' && {
+							$filter: {
+								is_directly_accessible_by__user: {
+									$any: {
+										$alias: 'dau',
+										$expr: {
+											1: 1,
+										},
 									},
 								},
 							},
-						},
+						}),
 						$orderby: 'app_name asc',
 					},
-					options,
+					options ?? {},
```

Comment:
`??` is a native JS operator
See: https://devdocs.io/javascript/operators/nullish_coalescing_operator
so it just depends what es version we are targeting in our tsconfig. Atm it will indeed emit two operations but in general we are trying (in all repos) to use that over `||` so b/c of its explicitness on not hiding errors when falsy values are passed (like `??` does. Eg passing `'test'` as options would error, while `''` would work, but that's not our intention, so `??` is better).